Sources/Momento/CacheClient.swift

```diff
@@ -354,7 +354,7 @@ public class CacheClient: CacheClientProtocol {
             return DeleteCacheResponse.error(DeleteCacheError(error: err))
         } catch {
             return DeleteCacheResponse.error(
-                DeleteCacheError(error: UnknownError(message: "unexpected error: \(error)"))
+                DeleteCacheError(error: UnknownError(message: "unexpected error: \(error)", innerException: error))
```

Comment:
would you mind also adding a little bit of extra formatting around the string interpolation of the error? just some quotes or something, like e.g. `unexpected error: '\(error)'`.

also, do we know anything about the type of the `error` object here? is there some known base class in swift stdlib? If so, does that base class have any other useful things we might want to add to the string interpolation?

Reply:
The compiler only offers `localizedDescription` as a property.

The Error type in Swift is defined as:
```
public protocol Error : Sendable {
}
```

Described as:

> A type representing an error value that can be thrown.
> 
> Any type that declares conformance to the `Error` protocol can be used to
> represent an error in Swift's error handling system. Because the `Error`
> protocol has no requirements of its own, you can declare conformance on
> any custom type you create.

Reply:
okay thanks!